llvm/test/TableGen/arm-preprocess.td

```diff
@@ -0,0 +1,7 @@
+// Check that parsing the preprocessor output creates exactly the same records
```

Comment:
Can it use some shared file instead? Something from llvm/include, maybe Target.td?

Reply:
Sure, done. Target.td is still complicated enough to make a good test.